### PR TITLE
update to tag element definitions

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: ebml_matroska.xml [hunk 1]
```diff
@@ -862,19 +862,66 @@
...
-    <documentation lang="en">Contain all UIDs where the specified meta data apply. It is empty to describe everything in the Segment.</documentation>
+    <documentation lang="en">Contain all UIDs that relate the specified metadata of the Tag to what the Tag is described. If empty or not present, then the Tag describes everything in the Segment.</documentation>
   </element>
   <element name="TargetTypeValue" path="0*1(\Segment\Tags\Tag\Targets\TargetTypeValue)" cppname="TagTargetTypeValue" id="0x68CA" type="uinteger" maxOccurs="1" minver="1" webm="0" default="50">
-    <documentation lang="en">A number to indicate the logical level of the target (see <a href="https://www.matroska.org/technical/specs/tagging/index.html#targettypes">TargetType</a>).</documentation>
+    <documentation lang="en">A number to indicate the logical level of the target.</documentation>
+    <restriction>
+      <enum value="70" label="COLLECTION">
+        <documentation lang="en" type="definition">The highest hierarchy of many different lower items.</documentation>
```
Comment (retokromer) [inline ebml_matroska.xml]: The «highest of lower» sounds pleonastic to me.

File: ebml_matroska.xml [hunk 2]
```diff
@@ -862,19 +862,66 @@
...
+    <documentation lang="en">Element containing metadata describing Tracks, Editions, Chapters, Attachments, or the Segment as a whole. A list of valid tags can be found <a href="https://www.matroska.org/technical/specs/tagging/index.html">here.</a></documentation>
   </element>
   <element name="Tag" path="1*(\Segment\Tags\Tag)" id="0x7373" type="master" minOccurs="1" minver="1" webm="0">
-    <documentation lang="en">Element containing Elements specific to Tracks/Chapters.</documentation>
+    <documentation lang="en">A single metadata descriptor.</documentation>
   </element>
   <element name="Targets" path="1*1(\Segment\Tags\Tag\Targets)" cppname="TagTargets" id="0x63C0" type="master" minOccurs="1" maxOccurs="1" minver="1" webm="0">
-    <documentation lang="en">Contain all UIDs where the specified meta data apply. It is empty to describe everything in the Segment.</documentation>
+    <documentation lang="en">Contain all UIDs that relate the specified metadata of the Tag to what the Tag is described. If empty or not present, then the Tag describes everything in the Segment.</documentation>
```
Comment (mbunkus) [inline ebml_matroska.xml]: "…to what the Tag is described" makes no sense to me (as a non-native English speaker). As a long-time Matroska developer I know what `Targets` does, but I image someone unfamiliar with Matroska will be very confused when reading this.

Alternative suggestions: "Contains all UIDs that this tag applies to." or "Specifies which other elements the metadata represented by the Tag applies to."

File: ebml_matroska.xml [hunk 3]
```diff
@@ -862,19 +862,66 @@
...
+        <documentation lang="en" type="definition">When an album or episode has different logical parts.</documentation>
+      </enum>
+      <enum value="30" label="TRACK / SONG / CHAPTER">
+        <documentation lang="en" type="definition">The common parts of an album or movie.</documentation>
+      </enum>
+      <enum value="20" label="SUBTRACK / PART / MOVEMENT / SCENE">
+        <documentation lang="en" type="definition">Corresponds to parts of a track for audio (like a movement).</documentation>
+      </enum>
+      <enum value="10" label="SHOT">
+        <documentation lang="en" type="definition">The lowest hierarchy found in music of movies.</documentation>
```
Comment (mbunkus) [inline ebml_matroska.xml]: "music *or* movies"? I guess you simply copy-pasted from another part of the spec here.

Comment (dericed) [inline ebml_matroska.xml]: fixed locally

File: ebml_matroska.xml (hunk 2 above)
Comment (dericed) [inline ebml_matroska.xml]: Updated to
> Specifies which other elements the metadata represented by the Tag applies to.

File: ebml_matroska.xml (hunk 1 above)
Comment (dericed) [inline ebml_matroska.xml]: updated to
> The highest hierarchical level that tags can describe.

**Comment by dericed** [comment]
I've updated the pull request according to @retokromer  and @mbunkus' suggestions. Please re-review.

**Comment by retokromer** [comment]
LGTM

**Pull request merged**